gen/_template/client.tmpl

```diff
@@ -4,6 +4,20 @@
 
 {{- if $.PathsClientEnabled }}
 {{- with $ops := $.Operations }}
+
```

Comment:
Could you split template change and autogenerated code into two commits, as [contributing guide suggests](https://github.com/ogen-go/ogen/blob/main/CONTRIBUTING.md#committing-generated-code)?

Reply:
Sure 👍 Didn't see that document, my bad. Now they are split.